azure-proto-network/NetworkInterface.cs

```diff
@@ -2,6 +2,9 @@
 
 namespace azure_proto_network
 {
+    /// <summary>
+    /// A class that represents a network interface in a resource group and the operatiosn that can be performed on it.
+    /// </summary>
     public class NetworkInterface : NetworkInterfaceOperations
     {
         internal NetworkInterface(AzureResourceManagerClientOptions options, NetworkInterfaceData resource)
```

Comment:
we have been documenting internal as well.

Reply:
Why?  These won't show up in intellisense.